bin/pulsar-managed-ledger-admin

```diff
@@ -20,11 +20,19 @@
 
 import argparse
 import traceback
-from google.protobuf.text_format import Merge
-from google.protobuf.text_format import MessageToString
 import sys
-    
-from proto import MLDataFormats_pb2
+try:
+    from google.protobuf.text_format import Merge
+    from google.protobuf.text_format import MessageToString
+except Exception as missingLib:
+    sys.exit("You need python protobuf library. See: https://github.com/google/protobuf/tree/master/python")
```

Comment:
You should be able to get that from PyPI as well, right? 
`pip install protobuf`